libs/remix-ui/workspace/src/lib/components/file-explorer-menu.tsx

```diff
@@ -10,27 +10,33 @@ export const FileExplorerMenu = (props: FileExplorerMenuProps) => {
     menuItems: [
       {
         action: 'createNewFile',
-        title: 'Create New File',
+        title: 'Create new file',
```

Comment:
I think you should use names from apps/remix-ide/src/app/tabs/locales/en/filePanel.json

Reply:
ok

Reply:
actually that is being used for `defaultMessage` , tooltip text is coming for fielPanel.json:
`tooltipText={<FormattedMessage id={filePanel.${action}} defaultMessage={title} />}`